accepted/2021/experimental-package-shading.md

```diff
@@ -0,0 +1,59 @@
+# Experiment: NuGet Package Shading
+
+(intro and scenarios to be added)
+
+## Usage
+
+### Package authors
+
+A *package author* may enable shading on any package reference as long as its assets are configured such that it is effectively private. Any other package references that have a shaded package reference as a transitive dependency must also be shaded. When the project is restored, shaded copies of the packages are created and swapped in. When the project is packed, the shaded assets are embedded in the resulting package.
+
```

Comment:
Can we introduce the term `shading` in the intro section?

Reply:
> Any other package references that have a shaded package reference as a transitive dependency must also be shaded.

I was lost here. Can we have examples?

Reply:
I rewrote this section; hopefully it's clearer? I don't want to complicate this section too much. Maybe I could just omit that detail and say "(subject to constraints detailed later in this spec)."

I will try to add a more detailed example with a diagram elsewhere.

Reply:
I guess an example will make it clear.

Reply:
Added examples with diagrams and an intro.